### PR TITLE
Updates to InputMirror and CharsRemaining delegators.

### PR DESCRIPTION
**Pull request by davywentworth**

### CODE REVIEW

File: js/Source/Delegators/Delegator.InputMirror.js [hunk 1]
```diff
@@ -34,9 +39,17 @@ Delegator.register(['change','keyup'], {
         var targetElement = element.getElement(target.selector);
         if (!targetElement) api.warn('Unable to find element for inputMirror selector: '+target.selector);
         if (targetElement && targetElement != element){
-          targetElement.set(target.property || 'value', element.get('value'));
+          var value = element.get('value');
+          if (api.get('stripHTML')){
+            value = value.replace(/</g, '&#60;').replace(/>/g, '&#62;').replace(/\//g, '&#47;');
+          }
```
Comment (anutron) [inline js/Source/Delegators/Delegator.InputMirror.js]: Given that this doesn't actually strip HTML (that would be `value.stripTags()`) I'd name this and the option "allowHTML". You could also consider adding support for `stripHTML` which would remove all of them (so `<b>foo</b>` becomes `foo`).

Also, generally speaking JS style doesn't use curly braces if it's a one liner:

`if (foo) bar();`

Use braces if it's multi-line or if there's an else that is a multiline:

````
if (foo) bar();
else baz();

if (foo){
  bar();
} else {
  baz();
  biz();
}
```

File: js/Source/Delegators/Delegator.InputMirror.js [hunk 2]
```diff
@@ -34,9 +39,17 @@ Delegator.register(['change','keyup'], {
...
+          if (api.get('stripNewlines')){
+            value = value.replace(/\n/g, '');
+          }
```
Comment (anutron) [inline js/Source/Delegators/Delegator.InputMirror.js]: 1 liner again

File: js/Source/Delegators/Delegator.InputMirror.js [hunk 3]
```diff
@@ -34,9 +39,17 @@ Delegator.register(['change','keyup'], {
         var targetElement = element.getElement(target.selector);
         if (!targetElement) api.warn('Unable to find element for inputMirror selector: '+target.selector);
         if (targetElement && targetElement != element){
-          targetElement.set(target.property || 'value', element.get('value'));
+          var value = element.get('value');
+          if (api.get('stripHTML')){
+            value = value.replace(/</g, '&#60;').replace(/>/g, '&#62;').replace(/\//g, '&#47;');
```
Comment (anutron) [inline js/Source/Delegators/Delegator.InputMirror.js]: You don't need to replace the `>` - just the `<`.

Comment (anutron) [inline js/Source/Delegators/Delegator.InputMirror.js]: And not the slash either;

File: js/Source/Delegators/Delegator.InputMirror.js (hunk 1 above)
Comment (marcferna) [inline js/Source/Delegators/Delegator.InputMirror.js]: we might want to do strip scripts? http://mootools.net/core/docs/1.5.2/Types/String#String:stripScripts

File: js/Source/Delegators/Delegator.InputMirror.js (hunk 3 above)
Comment (davywentworth) [inline js/Source/Delegators/Delegator.InputMirror.js]: from using it, it appears that i do need to replace these. if i don't, you can close the tag that this value is appearing in.

Comment (anutron) [inline js/Source/Delegators/Delegator.InputMirror.js]: Huh? I don't get it.

**Pull request closed without merging**